packages/oui-inline-adder/src/inline-adder-item.html

```diff
@@ -0,0 +1,18 @@
+<div class="oui-inline-adder__item">
+    <div class="oui-inline-adder__rows_container" ng-transclude></div>
+    <div class="oui-inline-adder__actions-container">
+        <oui-spinner ng-if="$ctrl.isLoading"></oui-spinner>
```

Comment:
We should add `size="s"` for the spinner.
See here https://master.ui-kit.ovh/#!/ovh-ui-kit/spinner

Reply:
This is done :)